vspace/src/main/webapp/WEB-INF/views/staff/modules/module.html

```diff
@@ -312,7 +356,30 @@ <h4 class="modal-title" id="deleteModalTitle">Confirm Deletion?</h4>
 		        </div>
 		    </div>
 		</div>
+		<div id="deleteModuleAlert" class="modal fade" role="dialog" data-value="0">
+		    <div class="modal-dialog">
+		        <div class="modal-content">
+		            <div class="modal-header">
+		                <h4 class="modal-title" id="deleteModalTitle">Confirm Deletion?</h4>
+		                <button type="button" id="closeSlide" class="close"
+		                    data-dismiss="modal" aria-hidden="true"><span class="icon-circle-close"></span></button>
+		            </div>
+		            <div class="modal-body">
+		                <p>
+		                    Are you sure you want to delete this module?
```

Comment:
Should say:
Are you sure you want to delete this module? All content will be deleted and all module links to this module will not work anymore.

Reply:
Done